yarn.lock

```diff
@@ -2155,7 +2155,11 @@ d3-scale@1.0.7:
     d3-time "1"
     d3-time-format "2"
 
-d3-selection@1, d3-selection@1.2.0, d3-selection@^1.1.0:
+d3-selection@1, d3-selection@^1.1.0:
+  version "1.3.0"
```

Comment:
I've filed d3/d3#3256 for fixing D3 so it doesn't pull in duplicates of the same package but at different versions.